File: Iniciante/1012 - Área.py
"""
Autor: Pierre Vieira
Data da submissão: 19/01/2019 14:58:55
"""
linha1 = input().split(" ")
a,b,c = linha1
triangulo = float(float(a)*float(c))/2
circulo = float(3.14159*float(c)*float(c))
trapezio = float(float(float(a)+float(b))*float(c))/2
quadrado = float(b)**2
retangulo = float(float(a)*float(b))
print("TRIANGULO: {:.3f}\nCIRCULO: {:.3f}\nTRAPEZIO: {:.3f}\nQUADRADO: {:.3f}\nRETANGULO: {:.3f}".format(triangulo, circulo, trapezio, quadrado, retangulo))

File: Java/Matematica/1012 - Área.java
import java.util.Scanner;

/*
 * Autor: Pierre Vieira
 * Data de submissão: 02/02/2020 14:30:25
 */
public class Main {

    public static void main(String[] args) {
        String[] entrada;
        Scanner teclado = new Scanner(System.in);
        double a, b, c, area_triangulo, area_circulo, area_trapezio, area_quadrado, area_retangulo;
        entrada = teclado.nextLine().split(" ");
        a = Double.parseDouble(entrada[0]);
        b = Double.parseDouble(entrada[1]);
        c = Double.parseDouble(entrada[2]);
        area_triangulo = a*c/2;
        area_circulo = 3.14159*c*c;
        area_trapezio = (a + b)*c/2;
        area_quadrado = b*b;
        area_retangulo = a*b;
        System.out.printf("TRIANGULO: %.3f\nCIRCULO: %.3f\nTRAPEZIO: %.3f\nQUADRADO: %.3f\nRETANGULO: %.3f\n", area_triangulo, area_circulo, area_trapezio, area_quadrado, area_retangulo);
    }
}
